src/Lykke.Service.FeeCalculator.AzureRepositories/CashoutFee/CashoutFeesRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AzureStorage;
+using Lykke.Service.FeeCalculator.Core.Domain.CashoutFee;
+
+namespace Lykke.Service.FeeCalculator.AzureRepositories.CashoutFee
+{
+    public class CashoutFeesRepository : ICashoutFeesRepository
+    {
+        private readonly INoSQLTableStorage<CashoutFeeEntity> _tableStorage;
+
+        public CashoutFeesRepository(INoSQLTableStorage<CashoutFeeEntity> tableStorage)
+        {
+            _tableStorage = tableStorage;
+        }
+        
+        public async Task<IEnumerable<ICashoutFee>> GetAllAsync()
+        {
+            return await _tableStorage.GetDataAsync(CashoutFeeEntity.GeneratePartitionKey());
```

Comment:
async await are redundant in all methods

Reply:
it returns IEnumerable<ICashoutFee>, not IEnumerable<CashoutFeeEntity>. Changed for Delete methods